source/operations/install-deploy-manage/multi-site-replication.rst

```diff
@@ -88,6 +88,11 @@ Site Healing
 
 Any MinIO deployment in the site replication configuration can resynchronize damaged :ref:`replica-eligible data <minio-site-replication-what-replicates>` from the peer with the most updated ("latest") version of that data.
 
+.. versionchanged:: RELEASE.2023-07-18T17-49-40Z
+
+   Site replication operations retry up to three (3) times.
+   Failed operations wait for the MinIO Scanner to identify the replication task again before requeueing for another three attempts.
```

Comment:
Same general idea about putting the onus on the scanner vs an operation (e.g. PUT or GET) that waits.

Reply:
Duplicated the updated warning from the bucket replication page.